src/components/Routes/Routes.test.tsx

```diff
@@ -17,12 +17,7 @@ jest.mock('react-router-dom', () => ({
   useParams: jest.fn(),
 }))
 
-describe('Routes', () => {
```

Comment:
This file was refactored to eliminate the nesting. This change aids in troubleshooting when there are problems with these tests and, in my opinion, is cleaner now.